patches/build-mac-tweak_info_plist.py.patch

```diff
@@ -1,24 +1,35 @@
 diff --git a/build/mac/tweak_info_plist.py b/build/mac/tweak_info_plist.py
-index 9ea794b1517fa2e65ac2c6a38b3176ec52a7b544..353081e3035cdd0d4dc5ebf347dbdb5ab8ef48c7 100755
+index 9ea794b1517fa2e65ac2c6a38b3176ec52a7b544..b6c7ef2f4b64347bbebdb809e980ff4ccdfea996 100755
 --- a/build/mac/tweak_info_plist.py
 +++ b/build/mac/tweak_info_plist.py
-@@ -248,6 +248,9 @@ def Main(argv):
+@@ -248,6 +248,13 @@ def Main(argv):
        default=True, help='Add SCM metadata [1 or 0]')
    parser.add_option('--branding', dest='branding', action='store',
        type='string', default=None, help='The branding of the binary')
-+  # BRAVE(shong): Add channel support
++  # BRAVE(shong): Add brave options
```

Comment:
can we create our own script to make these changes? I'd rather do a second round of "tweaking" than patch the script

Reply:
Good idea! I'll try.

Reply:
Done.